### ISSUE TITLE
BatchNorm2d in GNN

### ISSUE THREAD

**Issue by vlivashkin** [zhengdao-chen/GNN4CD#4]
Hi! Here in GNN baseline model https://github.com/zhengdao-chen/GNN4CD/blob/master/src/models.py#L93 BatchNorm2d is failing to perform 2d input.
Did you mean here BatchNorm1d?

**Comment by zhengdao-chen** [comment]
Hello! Thanks for bringing this up. I think it could be due to the version differences in PyTorch. I tried the original code with PyTorch=1.4.0, and indeed there was an issue with BatchNorm2d. After switching to BatchNorm1d the issue went away. I will try to update the code soon.
